views/articles/new.ejs

```diff
@@ -0,0 +1,41 @@
+<%- include ("../partials/header") %>
```

Comment:
Preview update is slow, works only when user wants to submit article

Reply:
Ah I see, that is due to the fact that "onchange" for the textview is only firing when you click off/on the box. I'll fix this.